tests/pipeline/test_all.nf.test

```diff
@@ -14,7 +14,44 @@ nextflow_pipeline {
         }
 
         then {
-            assert workflow.success
+            assertAll(
+                { assert workflow.success },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
+                { assert new File("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml").exists() },
+
+                { assert snapshot(path("$outputDir/db_tables/alleles.tsv"),
+                                path("$outputDir/db_tables/conditions_alleles.tsv"),
+                                path("$outputDir/db_tables/conditions.tsv"),
+                                path("$outputDir/db_tables/entities_proteins.tsv"),
+                                path("$outputDir/db_tables/entities.tsv"),
+                                path("$outputDir/db_tables/microbiomes_entities.no_weights.tsv"),
+                                path("$outputDir/db_tables/microbiomes_entities.nucl.tsv"),
+                                path("$outputDir/db_tables/microbiomes_entities.tsv"),
+                                path("$outputDir/db_tables/microbiomes.tsv"),
+                                path("$outputDir/db_tables/proteins_peptides.tsv"),
+                                path("$outputDir/db_tables/peptides.tsv.gz"),
+                                path("$outputDir/db_tables/proteins.tsv.gz"),
+                                path("$outputDir/db_tables/predictions.tsv.gz"),
+                                path("$outputDir/db_tables/stats.txt")).match("db_tables") },
+
+                { assert snapshot(path("$outputDir/entrez_data/microbiomes_entities.entrez.tsv")).match("entrez_data") },
+                { assert new File("$outputDir/entrez_data/entities_proteins.entrez.tsv").exists() },
+                { assert new File("$outputDir/entrez_data/proteins.entrez.tsv.gz").exists() },
+                { assert new File("$outputDir/entrez_data/taxa_assemblies.tsv").exists() },
+
+                { assert new File("$outputDir/figures/entity_binding_ratios.A_01_01.pdf").exists() },
+                { assert new File("$outputDir/figures/entity_binding_ratios.with_points.A_01_01.pdf").exists() },
+                { assert new File("$outputDir/figures/prediction_score_distribution.A_01_01.pdf").exists() },
+
+                { assert snapshot(path("$outputDir/logs/prediction_warnings.log"),
+                                path("$outputDir/logs/SUCCEEDED_unify_peptide_lengths.log"),
+                                path("$outputDir/logs/download_proteins.log")).match("logs") },
+
+                { assert new File("$outputDir/prodigal/proteins.pred_1.bin.1.tsv.gz").exists() },
+                { assert new File("$outputDir/prodigal/proteins.pred_1.bin.2.tsv.gz").exists() },
+                { assert new File("$outputDir/prodigal/proteins.pred_2.tsv.gz").exists() }
+            )
+
```

Comment:
```suggestion
```
I dont like so many blank lines. Above it might make sense to have blocks, but here it doesnt imho. There are more blank lines that could be removed. But thats not important ofc.